### PR TITLE
Set Caluma-Companion header

### PR DESCRIPTION
**Pull request by open-dynaMIX**
In order to configure visibilities for caluma-interval, we set a custom header:

"Caluma-Companion: interval"

### CODE REVIEW

**Comment by open-dynaMIX** [comment]
cc/ @kaldras

File: README.md [hunk 1]
```diff
@@ -215,6 +215,21 @@ class FormIntervalValidation(BaseValidation):
...
+
+Make sure, that caluma-interval has access to `Forms`, `Documents` and `Cases`.
+
+```python
+class MyVisibility(BaseVisibility):
+    @filter_queryset_for(Node)
+    def filter_queryset(self, node, queryset, info):
+        if info.context.META.get(
+            "HTTP_CALUMA_COMPANION", None
+        ) == "interval" and node in [Form, Document, Case]:
```
Comment (winged) [inline README.md]: I'm not sure we should publish an insecure example like this. If someone copy/pastes this in production, anybody could put a header like this in their requests and see everything.

Instead, I'd do proper auth and check the user's role. Or alternatively, implement a preshared-secret type auth

File: caluma_interval/client.py [hunk 2]
```diff
@@ -58,7 +58,11 @@ def execute(self, query, variables=None):
 
     def _send(self, query, variables):
         data = {"query": query, "variables": variables}
-        headers = {"Accept": "application/json", "Content-Type": "application/json"}
+        headers = {
+            "Accept": "application/json",
+            "Content-Type": "application/json",
+            "Caluma-Companion": "interval",
```
Comment (winged) [inline caluma_interval/client.py]: As said above, I'd prefer a preshared secret instead of just a "guessable" value

**Comment by open-dynaMIX** [comment]
@winged Thank you for stopping this madness! This is obviously a very bad idea. Sorry for wasting your time!

**Pull request closed without merging**